dist/stat_data.py

```diff
@@ -599,6 +599,9 @@ def __init__(self, name, desc, flags=''):
     TxnStat('txn_read_queue_walked', 'read timestamp queue entries walked'),
     TxnStat('txn_rollback', 'transactions rolled back'),
     TxnStat('txn_rollback_hs_removed', 'rollback to stable updates removed from history store'),
+    TxnStat('txn_rollback_keys_removed', 'rollback to stable keys removed'),
+    TxnStat('txn_rollback_keys_restored', 'rollback to stable keys restored'),
+    TxnStat('txn_rollback_pages_visited', 'rollback to stable pages visited'),
```

Comment:
I hate to see these get too long, but don't we have to distinguish between normal transactions being rolled back and stuff being done by rollback to stable? I think the rollback-to-stable statistics need a common prefix, otherwise it gets confusing.

I could live with `txn_rts_XXX`, maybe?

I won't die on this hill if others disagree.

Reply:
This is a very good point. I can live with `rts` also. This would need other nearby statistics to be renamed internally. It doesn't need to change the text description.

Reply:
I'm happy to make the change, or not make the change. @kommiharibabu what do you think?

Reply:
I agree with @sueloverso and @keithbostic. `rts` is a better representation than `rollback`. Please do the changes.

Reply:
Changes made.